src/components/Policy/Table/ExpandedContent/Dates.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { Text } from '@patternfly/react-core';
+import { format } from 'date-fns';
+
+interface DateProps {
+    updated: Date;
+    created: Date;
+}
+
+const dateFormatString = 'dd MMM yyyy';
```

Comment:
Is that consistent with other apps - given that the US likes that (to me strange) mm/dd/yy format a lot

Reply:
I took this from the mock up, it yields e.g. `19 Feb 2020`